hoomd/md/EvaluatorPairOPP.h

```diff
@@ -167,6 +170,30 @@ class EvaluatorPairOPP
             }
         }
 
+    // TODO: update the derivatives for the parameters including Cs and eta(s)
+    DEVICE void alchemParams(const Scalar* alphas)
+        {
+        params.k *= alphas[0];
+        params.phi *= alphas[1];
+        }
+
+    DEVICE void evalAlchDerivatives(Scalar* alchemical_derivatives, const Scalar* alphas)
+        { // OPP without any modification at this point
+            {
+            Scalar r = fast::sqrt(rsq);
+            Scalar r1inv = Scalar(1.0) / r;
+            Scalar r3inv = r1inv * r1inv * r1inv;
+            // Scalar r15inv = r3inv * r3inv * r3inv * r3inv * r3inv;
+
+            alchemical_derivatives[0]
+                = -sin(params.k * alphas[0] * (r - Scalar(1.0)) + params.phi * alphas[1]) * params.k
+                  * (r - Scalar(1.0)) * r3inv;
+            alchemical_derivatives[1]
+                = -sin(params.k * alphas[0] * (r - Scalar(1.0)) + params.phi * alphas[1])
+                  * params.phi * r3inv;
+            }
+        }
+
```

Comment:
This code is remaining from an old implementation that only used k and phi as its params. It does not cover the current OPP methods and must be replaced.

Reply:
OPP is not exposed in Python, not used in publications (as far as I know), and would therefore require significant effort to enable. I am deleting this code for now, future work could enable this functionality when needed.